### ISSUE TITLE
Execute the code to contribute

### ISSUE THREAD

**Issue by rafa-as** [primefaces/primeng#4929]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
I can't reproduce this bug on plunkr

**Current behavior**
1. Fork repo
2. Install dependencies: `yarn install`
3. Execute start npm task: `yarn run start`
4. The execution throws the following error:

```
node ~/.yarn/bin/yarn.js run start
yarn run v1.3.2
$ ng serve
** NG Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
Date: 2018-01-23T08:06:43.627Z
Hash: fe2ae0a1b0bdd5f6c0a5
Time: 3762ms
chunk {inline} inline.bundle.js (inline) 5.79 kB [entry] [rendered]
chunk {main} main.bundle.js (main) 2.93 kB [initial] [rendered]
chunk {polyfills} polyfills.bundle.js (polyfills) 636 bytes [initial] [rendered]
chunk {scripts} scripts.bundle.js (scripts) 1.85 MB [initial] [rendered]
chunk {styles} styles.bundle.js (styles) 706 kB [initial] [rendered]
chunk {vendor} vendor.bundle.js (vendor) 853 kB [initial] [rendered]

ERROR in TypeError: _this.host.getOutputName is not a function
    at getOriginalName (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:29582:32)
    at ReferenceTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:29627:43)
    at visitValue (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:426:24)
    at primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:65
    at Array.forEach (<anonymous>)
    at ReferenceTransformer.ValueTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:26)
    at ReferenceTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:29654:60)
    at visitValue (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:426:24)
    at primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:65
    at Array.forEach (<anonymous>)
    at ReferenceTransformer.ValueTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:26)
    at ReferenceTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:29654:60)
    at visitValue (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:426:24)
    at primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:65
    at Array.forEach (<anonymous>)
    at ReferenceTransformer.ValueTransformer.visitStringMap (primeng/node_modules/@angular/compiler/bundles/compiler.umd.js:483:26)

webpack: Failed to compile.
```

**Expected behavior**
The project has started normally without errors

**Minimal reproduction of the problem with instructions**
1. Fork repo
2. Install dependencies: `yarn install`
3. Execute start npm task: `yarn run start`
4. Wait to start and look the error

**What is the motivation / use case for changing the behavior?**
Contribute to project fixing some bugs

**Please tell us about your environment:**
_OS_: MAC OS X High Sierra 10.13.2
_IDE_: WebStorm 2018.1
_Package Manager_: Yarn 1.3.2
_Node_: 9.4.0

* **Angular version:** 5.2.1

* **PrimeNG version:** 5.2.0-rc.2-SNAPSHOT

* **Browser:** all
 
* **Language:** Typescript

* **Node (for AoT issues):** `node --version` = 9.4.0

**Comment by rafa-as** [comment]
I found this issue on angular core: https://github.com/angular/angular/issues/21621

**Comment by cagataycivici** [comment]
Does not seem PrimeNG related.

**Comment by rafa-as** [comment]
@cagataycivici I understand your response but my clone is a fresh copy of the original and I can't start it.
You have the same Angular version 5.2.1 ?

**Comment by rafa-as** [comment]
@cagataycivici sorry but effectively is a primeng bug.

All packages of angular needs to be the same version and some packages are fixed on the 5.0.0 and the others has the caret modifier, specifically the following:

- @angular/platform-browser
- @angular/platform-browser-dynamic
- @angular/language-service
- @angular/compiler-cli
- @angular/language-service

I'm going to open a new issue and the PR.
